app/services/laa_reference_unlinker.rb

```diff
@@ -10,9 +10,9 @@ def initialize(defendant_id:, user_name:, unlink_reason_code:, unlink_other_reas
   end
 
   def call
-    unlink_maat_reference!
     push_to_queue unless laa_reference.dummy_maat_reference?
     update_offences_on_common_platform
```

Comment:
Likewise here, will the code stop before reaching the unlink method if CP fails?